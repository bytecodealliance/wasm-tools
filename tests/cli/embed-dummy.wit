// RUN[legacy]: component embed -t --dummy-names legacy % | strip -t
// RUN[standard32]: component embed -t --dummy-names standard32 % | strip -t

package a:b;

interface x {
  y: func() -> string;
}

world w {
  import x;
  export x;
}
