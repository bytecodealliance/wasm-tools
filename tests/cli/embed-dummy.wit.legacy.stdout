(module
  (type (;0;) (func (param i32)))
  (type (;1;) (func (result i32)))
  (type (;2;) (func (param i32 i32 i32 i32) (result i32)))
  (type (;3;) (func))
  (import "a:b/x" "y" (func (;0;) (type 0)))
  (memory (;0;) 0)
  (export "a:b/x#y" (func 1))
  (export "cabi_post_a:b/x#y" (func 2))
  (export "memory" (memory 0))
  (export "cabi_realloc" (func 3))
  (export "_initialize" (func 4))
  (func (;1;) (type 1) (result i32)
    unreachable
  )
  (func (;2;) (type 0) (param i32))
  (func (;3;) (type 2) (param i32 i32 i32 i32) (result i32)
    unreachable
  )
  (func (;4;) (type 3))
)
